configs/apps/ufs/spack.yaml

```diff
@@ -0,0 +1,63 @@
+spack:
+  concretization: separately
+  view: false
+  include:
+  - config.yaml
+  - packages.yaml
+  - modules.yaml
+  - ${SPACK_STACK_DIR}/configs/repos/repos.yaml
+  - site.yaml
+
+ specs:
```

Comment:
This could be 'ufs-env` (from the `package.py`) to encapsulate the spec, and you could remove all these specs. 

Though, my preference is to specify the specs in a `spack.yaml`. I think the `*-env` packages are kind of a hacky solution to what environments offer.

```
    depends_on('eccodes', type=('build', 'run'))
     depends_on('py-setuptools', type=('build', 'run'))
     depends_on('py-cython', type=('build', 'run'))
     depends_on('py-numpy', type=('build', 'run'))
     depends_on('py-proj', type=('run'))
```
vs. 

```
specs:
- eccodes
- py-setuptools
- py-numpy
- py-proj
```

Reply:
So this means that each env is specific to an application? By doing `spack install` you would get whatever is needed for the app you requested, but you wouldn't be able install the missing packages for another app in a convenient way?

Reply:
Also good points. Each env would be specific to an application, yes. Though, I imagined some sort of combined env that contains 'everything' (kind of like `stack_noaa` from hpc-stack) along with trimmed down envs tailored to specific apps.